### ISSUE TITLE
Compatibility with Node.js 15

### ISSUE THREAD

**Issue by JannesMeyer** [jasmine/jasmine-npm#172]
When running `npm install jasmine` I noticed a warning about jasmine not being compatible with Node.js 15.0, which was released as current/stable on October 20th.

```
npm WARN EBADENGINE Unsupported engine {
npm WARN EBADENGINE   package: 'jasmine@3.6.3',
npm WARN EBADENGINE   required: { node: '^10 || ^12 || ^14' },
npm WARN EBADENGINE   current: { node: 'v15.0.1', npm: '7.0.3' }
npm WARN EBADENGINE }
```

As far as I can tell, there are no incompatibilities with Node 15.

Is this a case of not having updated the engine requirements yet or did you decide to only support LTS releases?

**Comment by sgravrock** [comment]
We only support LTS releases. The odd-numbered releases should work fine, but they don't live long enough for us to be able to commit to supporting them for an entire major release cycle.

**Comment by JannesMeyer** [comment]
Ok, thanks for your reply. That's good to know. I will close this issue then.

**Comment by jehon** [comment]
Hello,
What about differentiating "support" from "node requirements" ?
They are different for me, and the package.json engine requirements is quiet blocking...

**Comment by sgravrock** [comment]
I've removed the engines field from package.json.

**Comment by jehon** [comment]
For those coming to this place: version 3.6.3 does not have the fix, waiting for a new release.

**Comment by sgravrock** [comment]
I've just released 3.6.4 which contains this change.
